ang/civicase/activity/factories/format-activity.factory.js

```diff
@@ -1,21 +1,26 @@
 (function (angular, $, _, CRM) {
   var module = angular.module('civicase');
 
-  module.factory('formatActivity', function (ContactsCache, ActivityStatusType, ActivityStatus, ActivityType, CaseStatus, CaseType) {
+  module.factory('formatActivity', function (ContactsCache, ActivityStatusType,
+    ActivityStatus, ActivityType, CaseStatus, CaseType, isTruthy) {
     var activityTypes = ActivityType.getAll(true);
     var activityStatuses = ActivityStatus.getAll();
     var caseTypes = CaseType.getAll();
     var caseStatuses = CaseStatus.getAll();
 
     return function (act, caseId) {
-      act.category = (activityTypes[act.activity_type_id].grouping ? activityTypes[act.activity_type_id].grouping.split(',') : []);
+      act.category = (activityTypes[act.activity_type_id].grouping
+        ? activityTypes[act.activity_type_id].grouping.split(',')
+        : []);
       act.icon = activityTypes[act.activity_type_id].icon;
       act.type = activityTypes[act.activity_type_id].label;
       act.status = activityStatuses[act.status_id].label;
       act.status_name = activityStatuses[act.status_id].name;
       act.status_type = getStatusType(act.status_id);
       act.is_completed = act.status_type !== 'incomplete'; // FIXME doesn't distinguish cancelled from completed
-      act.is_overdue = (typeof act.is_overdue === 'string') ? (act.is_overdue === '1') : act.is_overdue;
+      act.is_overdue = (typeof act.is_overdue === 'string')
+        ? isTruthy(act.is_overdue)
+        : act.is_overdue;
```

Comment:
according to `isTruthy` usage there is not need to check if `is_overdue` is a string, just pass this directly to `isTruthy`